core/src/main/mima-filters/1.0.5.backwards.excludes

```diff
@@ -0,0 +1,3 @@
+# PR #831
```

Comment:
Just noticed that the file name is wrong. It should be `1.0.4.backwards.excludes`.

Reply:
done

Reply:
Am I right understand that I need 2 approvals from you and @2m